### ISSUE TITLE
Action Pad and Action - Provide valid, concise, and meaningful alternative text for image buttons - (2036569767)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7779]
### Violation: 
 Provide valid, concise, and meaningful alternative text for image buttons 

![image](https://github.com/Esri/calcite-design-system/assets/5023024/92f30649-5a3e-444f-ad5b-a9dedd730e82)

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Severity: 
 6 

### Media Type: 
 Forms 

--- 
### Areas for Remediation: 
- Title: Action Pad and Action - Provide valid, concise, and meaningful alternative text for image buttons - (2036569767) 
- Module: Action Pad and Action 
**Issue**
There are expand/collapse image buttons that do not provide the complete accessible names. The expand/collapse state is defined using an accessible name. Examples include:

- Action pad
- Action pad small scale
- Action pad expanded
- Action pad horizontal layout
- Action pad grid layout

**User Impact**
Without meaningful alternative text, any information conveyed by these image buttons will not be conveyed to screen reader users and they will be announced as unlabeled buttons.

**Code Reference** 
 ```
<button aria-busy="false" aria-disabled="false" aria-label="Expand" class="button">
  <div aria-hidden="true" class="icon-container">
    (...)
  </div>
  <div class="text-container">Expand</div>
</button>
 ``` 
**Suggestion**
Ensure that image buttons provide meaningful alternative text. This can be achieved by providing an aria-label for ``` <button> ``` elements.

**Compliant Code Example** 
 ```
<button aria-busy="false" aria-disabled="false" aria-label="Action pad Expand" class="button">
  <div aria-hidden="true" class="icon-container">
    (...)
  </div>
  <div class="text-container">Expand</div>
</button>
 ```
 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by driskull** [comment]
Seems like an app issue. Can use the label property here.

**Comment by geospatialem** [comment]
This has been mitigated prior to `2.0.0` with the `"expand-tooltip"` slot to provide specific context when the `action-pad` is collapsed to provide context on expansion, and also provides the `"collapse"` context. Both the `"expand"` and `"collapse"` strings can be modified by the developer for additional configuration.
